excel-importer/src/com/axonivy/util/excel/importer/ExcelReader.java

```diff
@@ -87,11 +89,42 @@ private static void updateColumn(Column column, Cell cell) {
         && !CellUtils.isInteger(cell)) {
       column.setType(Double.class);
     }
+    if (cell.getCellType() == CellType.STRING) {
+      column.setType(String.class);
+      column.setDatabaseFieldLength(DEFAULT_STRING_LENGTH);
+    }
     if (column.getType().equals(String.class)) {
-      var cellLength = cell.getStringCellValue().length();
-      if (cellLength > column.getDatabaseFieldLength()) {
-        column.setDatabaseFieldLength(cellLength);
+      var cellValue = getCellValueAsString(cell);
+      if (cellValue.length() > column.getDatabaseFieldLength()) {
+        column.setDatabaseFieldLength(cellValue.length());
+      }
+    }
+  }
+
+  public static String getCellValueAsString(Cell cell) {
+    if (cell == null) {
+      return StringUtils.EMPTY;
+    }
+    switch (cell.getCellType()) {
+    case STRING:
+      return cell.getStringCellValue();
+    case NUMERIC:
+      if (DateUtil.isCellDateFormatted(cell)) {
+        return cell.getDateCellValue().toString();
+      } else {
+        DecimalFormat decimalFormat = new DecimalFormat("#");
+        decimalFormat.setMaximumFractionDigits(0);
+        decimalFormat.setGroupingUsed(false);
+        return decimalFormat.format(cell.getNumericCellValue());
       }
+    case BOOLEAN:
+      return String.valueOf(cell.getBooleanCellValue());
+    case FORMULA:
+      return cell.getCellFormula();
+    case BLANK:
+      return StringUtils.EMPTY;
+    default:
+      return "Unsupported cell type";
```

Comment:
I think it's not nice to fill the customers data with such values; why don't we just omit it and handle it just like a "null" case?